### PR TITLE
v10 GA Release Notes

### PR DESCRIPTION
**Pull request by askdba**
Signed-off-by: Alkin Tezuysal <alkin.tezuysal@gmail.com>

Updated v10.0.0 Release Notes 

## Description
This version includes all changes since v10.0RC1 
## Checklist
- [ ] Tests were added or are not required
- [ ] Documentation was added or is not required

### CODE REVIEW

File: doc/releasenotes/10_0_0_release_notes.md [hunk 1]
```diff
@@ -187,19 +197,19 @@ The following PRs made changes to behaviors that clients might rely on. They sho
 * VTAdmin: [vtadmin-api] Add /vtexplain endpoint #7528
 * VTAdmin: [vtadmin-api] Reorganize tablet-related functions into vtadmin/cluster/cluster.go #7553
 * VTAdmin: Three small bugfixes in Tablets table around stable sort order, display type lookup, and filtering by type #7568
-* VTAdmin: [vtadmin] Add GetSchema endpoint #7596
-* VTAdmin: [vtadmin/testutil] Add testutil helper to manage the complexity of recursively calling WithTestServer #7601
-* VTAdmin: [vtadmin] Add FindSchema route #7610
-* VTAdmin: [vtadmin-web] Add simple /schema view with table definition #7615
+* VTAdmin:  [vtadmin] Add GetSchema endpoint #7596
+* VTAdmin:  [vtadmin/testutil] Add testutil helper to manage the complexity of recursively calling WithTestServer #7601
+* VTAdmin:  [vtadmin] Add FindSchema route #7610
```
Comment (shlomi-noach) [inline doc/releasenotes/10_0_0_release_notes.md:202]: spacing issues

File: doc/releasenotes/10_0_0_release_notes.md [hunk 2]
```diff
@@ -187,19 +197,19 @@ The following PRs made changes to behaviors that clients might rely on. They sho
...
+* [vtadmin-web] Add simple /schema view with table definition #7615
 * VTAdmin: [vtadmin] vschemas api endpoints #7625
 * VTAdmin: [vtadmin] Add support for additional service healthchecks in grpcserver #7635
-* VTAdmin: [vtadmin] test refactors #7641
+* VTAdmin:  [vtadmin] test refactors #7641
```
Comment (shlomi-noach) [inline doc/releasenotes/10_0_0_release_notes.md:206]: spacing

File: doc/releasenotes/10_0_0_release_notes.md [hunk 3]
```diff
@@ -187,19 +197,19 @@ The following PRs made changes to behaviors that clients might rely on. They sho
...
 * VTAdmin: [vtadmin] propagate error contexts #7642
 * VTAdmin: [vtadmin] tracing refactor #7649
 * VTAdmin: [vtadmin] GetWorkflow(s) endpoints #7662
 * VTAdmin: [vitessdriver|vtadmin] Support Ping in vitessdriver, use in vtadmin to healthcheck connections during Dial #7709
 * VTAdmin: [vtadmin]  Add to local example #7699
-* VTAdmin: vtexplain lock #7724
+* VTAdmin vtexplain lock #7724
```
Comment (shlomi-noach) [inline doc/releasenotes/10_0_0_release_notes.md:212]: missing `:`

**Review by shlomi-noach — CHANGES_REQUESTED** [review]

**Review by askdba — COMMENTED** [review]
Fixed

File: doc/releasenotes/10_0_0_release_notes.md [hunk 4]
```diff
@@ -81,6 +81,13 @@ The following PRs made changes to behaviors that clients might rely on. They sho
 * VTGate: Show columns query on system schema #7729
 * VTGate: Disallow foreign key constraint on ddl #7780
 * VTGate: VTGate: support -enable_online_ddl flag #7694
+* VTGate: Default to false for system settings to be changed per session at the database connection level #7921
+* VTGate: vtctl: return error on invalid ddl_strategy #7924
+* VTGate: [10.0] Squashed backport of #7903 #7927
+* VTGate: [10.0] Fix bug with reserved connections to stale tablets #7935
+* VTGate: [10.0] Fix for keyspaces_to_watch regression #7936
+* VTGate:  [10.0] Update healthy tablets correctly for primary down #7937
```
Comment (shlomi-noach) [inline doc/releasenotes/10_0_0_release_notes.md:89]: spacing

File: doc/releasenotes/10_0_0_release_notes.md [hunk 5]
```diff
@@ -187,8 +197,8 @@ The following PRs made changes to behaviors that clients might rely on. They sho
 * VTAdmin: [vtadmin-api] Add /vtexplain endpoint #7528
 * VTAdmin: [vtadmin-api] Reorganize tablet-related functions into vtadmin/cluster/cluster.go #7553
 * VTAdmin: Three small bugfixes in Tablets table around stable sort order, display type lookup, and filtering by type #7568
-* VTAdmin: [vtadmin] Add GetSchema endpoint #7596
-* VTAdmin: [vtadmin/testutil] Add testutil helper to manage the complexity of recursively calling WithTestServer #7601
+* VTAdmin:  [vtadmin] Add GetSchema endpoint #7596
+* VTAdmin:  [vtadmin/testutil] Add testutil helper to manage the complexity of recursively calling WithTestServer #7601
```
Comment (shlomi-noach) [inline doc/releasenotes/10_0_0_release_notes.md:201]: spacing

**Review by shlomi-noach — CHANGES_REQUESTED** [review]

**Review by shlomi-noach — APPROVED** [review]

**Pull request merged**